codecparsers/h265Parser.cpp

```diff
@@ -220,15 +249,15 @@ SharedPtr<PPS> Parser::getPps(uint8_t id) const
 bool Parser::profileTierLevel(ProfileTierLevel* ptl, NalReader& nr,
     uint8_t maxNumSubLayersMinus1)
 {
-    ptl->general_profile_space = nr.read(2);
-    ptl->general_tier_flag = nr.read(1);
-    ptl->general_profile_idc = nr.read(5);
+    READ_BITS(nr, ptl->general_profile_space, 2);
+    READ(nr, ptl->general_tier_flag);
+    READ_BITS(nr, ptl->general_profile_idc, 5);
     for (uint32_t j = 0; j < 32; j++)
-        ptl->general_profile_compatibility_flag[j] = nr.read(1);
-    ptl->general_progressive_source_flag = nr.read(1);
-    ptl->general_interlaced_source_flag = nr.read(1);
-    ptl->general_non_packed_constraint_flag = nr.read(1);
-    ptl->general_frame_only_constraint_flag = nr.read(1);
+        READ(nr, ptl->general_profile_compatibility_flag[j]);
+    READ(nr, pptl->general_progressive_source_flag);
```

Comment:
@xuguangxin oh, it is indeed wrong. I use ./configure --help and it says h265dec is default on.